application/templates/admin/data_sources.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+{% from "_shared/_breadcrumb.html" import breadcrumb %}
+
+{% set breadcrumbs =
+  [
+    {"url": url_for("static_site.index"), "text": "Ethnicity facts and figures"},
+    {"url": url_for('admin.index'), "text": "Admin"},
+  ]
+%}
+
+{% block pageTitle %}{% if q %}{{ q }} - Search data sources{% else %}Data sources{% endif %}{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Data sources</h1>
+
+
+      <form method="GET" action="{{ url_for('admin.data_sources' )}}">
```

Comment:
This should probably be using a WTForm SearchForm in `application.cms.forms` just to keep it clear that's how we do forms. And then would be good to also have CSRF protection, even though it's a non-destructive form.

Reply:
If we have CSRF then it'd be impossible to link to a search results page?

Reply:
Yeah, sorry, my brain isn't working right clearly. We don't want CSRF obviously. Missed that it was just a GET.

Reply:
(Would still be nice to use a wtform for the input though)

Reply:
What does WTForm buy us in this case? The other search form doesn't use it, so we should probably be consistent...?

Reply:
WTForms is also how we render inputs consistently; it buys us automatically managing the presentation of these fields according to our GOV.UK Design System setup. It buys us not duplicating this logic in multiple places and instead being able to use template partials/macros.

The other search form arguably should also be a WTForm. Probably just the same one, to be honest: `DataSourceSearchForm`.

Reply:
Good point. Done in 47c7b8f590b85b47c7fb92616b4e4db69951b9b1